.circleci/config.yml

```diff
@@ -33,10 +35,10 @@ jobs:
             cd login-workflow
             yarn lint
       - run:
-          name: Tests
+          name: Unit Tests
           command: |
             cd login-workflow
-            yarn test:ci
+            yarn test --no-watch --code-coverage
```

Comment:
We should be good remove the `yarn test:ci` script from the package.json right? I don't think it's used in any other place.

Reply:
just removed it

Reply:
Looks good to me!